pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -864,6 +869,16 @@ func (oe *operationExecutor) generateUnmountDeviceFunc(
 				unmountDeviceErr)
 		}
 
+		inUseErr := util.CheckDeviceInUseBySystem(deviceToDetach.DevicePath)
+		if inUseErr != nil {
+			// On failure, return error. Caller will log and retry.
+			return fmt.Errorf(
+				"OpenFile to verify unmount failed for volume %q (spec.Name: %q) with: %v",
```

Comment:
@ciwang @saad-ali This error is ambiguous. Hence why having two return values. Did the open call fail? Or did it just say EBusy? Another error is worth reporting.